README.md

```diff
@@ -25,16 +25,55 @@ git clone https://github.com/aerogear/mobile-developer-console && cd mobile-deve
 4. Log in with user `developer` (developer/123)
 5. Follow the link to mobile-developer-console
 
-### Full experience (Use APB to deploy mobile-developer-console)
+### Full experience
+
+#### Spin up OpenShift cluster locally
+
+**(OpenShift 3.9) Use mobile-core installer**
```

Comment:
Should we mention mobile-core? Will not it be deprecated?

Reply:
I think let's keep it for now if MDC still works with mobile-core (OS 3.9) and let's remove it once we decide to deprecate mobile-core. Wdyt?